test/test_utils.ml

```diff
@@ -137,6 +137,50 @@ let util_safe_html _ =
     (Adef.safe {|<a href="localhost:2318/foo_w?lang=fr&#38;image=on">foo</a>|})
     (Util.safe_html {|<a href="localhost:2318/foo_w?lang=fr&image=on">foo</a>|})
 
+let util_eval_translate _ =
+  let conf = Config.empty in
+  let conf = { conf with env = ("lang", Adef.encoded "fr") :: conf.env } in
+  let _ = Hashtbl.add conf.lexicon "of%s" "d[e |']%s" in
+  let _ = Hashtbl.add conf.lexicon "of%s %s" "d[e |']%s %s" in
+  let _ = Hashtbl.add conf.lexicon "male/female/neuter" "homme/femme/neutre" in
+  let _ = Hashtbl.add conf.lexicon "half/half" "demi/demie" in
+  let _ = Hashtbl.add conf.lexicon "jhon/johann" "jean/jeanne" in
+  let _ =
+    Hashtbl.add conf.lexicon "%sbrother/%ssister/%sneuter"
+      "%sfrère/%ssoeur/%sfrère ou soeur"
+  in
+  let test aaa sup s c =
+    let bbb = Templ.eval_transl_lexicon conf sup s c in
+    if aaa <> bbb then Printf.eprintf "Res: %s <> %s\n" aaa bbb;
+    assert_equal aaa bbb
+  in
+  let test1 aaa s =
+    let bbb = Util.simple_decline conf s in
+    assert_equal aaa bbb
+  in
+  let test_start aaa s =
+    let bbb = if Util.start_with_vowel conf s then "1" else "0" in
+    assert_equal aaa bbb
```

Comment:
for alcotest:
you can change the `assert_equal aaa bbb` for `(check string) "" aaa bbb`
and it should be good